ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py

```diff
@@ -0,0 +1,59 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# prod
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_np.valid_float_dtypes),
+    dtype=st.sampled_from(ivy_np.valid_float_dtypes + (None,)),
+    where=np_frontend_helpers.where(),
+    as_variable=helpers.array_bools(),
```

Comment:
Please don't pass in anything for as_variable, native_array, container, with_out, instance_method inside given, since that is being handled by the @handle_cmd_line_args decorator, and it will only overwrite the values for these flags, which we don't want.

Reply:
By the way, we should now be using ivy.current_backend().valid_dtypes inside our helper functions, instead of ivy_np.valid_dtypes.

Reply:
Thank you, I have changed everything you mentioned here.